addons/transifex/models/models.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+
+class BaseModel(models.AbstractModel):
+    _inherit = 'base'
+
+    def get_field_translations(self, field_name, langs=None):
+        """ :return dict translations: [{lang: lang, source: val_en, value: val_lang, transifexUrl: transifex_url)] """
+        translations, context = super().get_field_translations(field_name, langs=langs)
+        self._update_transifex_url(translations)
+        return translations, context
+
+    def _update_transifex_url(self, translations):
+        """ Construct transifex URL based on the module on configuration """
+        self.ensure_one()
+        # .tx/config files contains the project reference
+        # using ini files like '[odoo-master.website_sale]'
+        external_id = self.get_external_id().get(self.id)
+        if not external_id:
+            return
+        module = external_id.split('.')[0]
+        if not module:
+            return
+
+        for translation in translations:
+            translation['module'] = module
+
+        TransifexTranslation = self.env['transifex.translation']
+        transifex_metadata = TransifexTranslation._get_transifex_metadata((module,), [translation['lang'] for translation in translations])
+        TransifexTranslation._update_transifex_url(transifex_metadata, translations)
```

Comment:
What is the purpose of making those two calls?  You wouldn't modify `transifex_metadata` before passing it to `TransifexTranslation._update_transifex_url()`, would you?  IMHO, the right API should be
```suggestion
        self.env['transifex.translation']._update_transifex_url(translations, (module,))
```